scripts/rename_resources.py

```diff
@@ -23,6 +23,12 @@
     for num in [1, 2]
 }
 
+resource = 'aws_cloudwatch_log_group'
+renamed.update({
+    f'{resource}.{name}': f'{resource}.{name}_api_gateway'
+    for name in ['indexer', 'service']
+})
+
```

Comment:
Please explain this to me in a PL slot.

Reply:
Decided in PL to not rename the resorces `aws_cloudwatch_log_group.indexer` and `aws_cloudwatch_log_group.service`.